astropy_helpers/sphinx/ext/utils.py

```diff
@@ -63,3 +67,111 @@ def find_mod_objs(modname, onlylocals=False):
         objs = [e for i, e in enumerate(objs) if valids[i]]
 
     return localnames, fqnames, objs
+
+
+def find_autosummary_in_lines(lines, module=None, filename=None):
```

Comment:
I'd suggest renaming this to something like `find_autosummary_in_lines_for_automodsumm`.  Just to make sure someone else looking at it doesn't think it's for more general use or a bug-fix of sphinx or something.